lib/ASTGen/Sources/ASTGen/LegacyParse.swift

```diff
@@ -0,0 +1,43 @@
+import ASTBridging
+import BasicBridging
+import SwiftSyntax
+import ParseBridging
+
+
+
+extension ASTGenVisitor {
+
+  private func bridgedSourceLocForParse(_ node: some SyntaxProtocol) -> BridgedSourceLoc {
+    return BridgedSourceLoc(at: node.position, in: self.base)
```

Comment:
I wonder if this should be `positionAfterSkippingLeadingTrivia` because that corresponds to the start location of the token in the C++ AST.

Reply:
It doesn't really matter, but you are right. I removed this method. and changed to use the existing `SyntaxProtocol.bridgedSourceLoc(in: ASTGenVisitor)`  method.